lasso.R

```diff
@@ -0,0 +1,55 @@
+library(glmnet) # load 'glmnet' package
+
+data <- readRDS("./Data/df_index_demographic_vars.RDS") # read index and demographics data provided by Carolina
```

Comment:
Where is this data? Can you show me a screenshot? For some reason, my branch does not have this data. If this is the data from @CarolinaVelasco's PR, I don't know how it is being used here since it hasn't been merged into Master. Sorry im so confused!

Reply:
This will be resolved when the data is in master